### PR TITLE
Create Ju5ts0cRypt0 3 projects

### PR DESCRIPTION
**Pull request by Ju5ts0cRypt0**
3 new projects Cardano Creatures, Cosmic Cannabis and SamRed

### CODE REVIEW

File: Ju5ts0cRypt0 3 projects [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            "f976d018b11cd9e7168c199344cbbd65d6b7e4dad81bb40908477714",
+            "b6831985e6385d9b20d0914e3a1864e0d8bf288072871d2ded1b9e18"
+        ]
+    },
+    {
+        "project": "Cosmic Cannabis",
+        "policies": [
+            "f1332030dc83f8dd4765819c34c65e41269e93c4895e4385f6342da2"
+        ]
+    }
```
Comment (weirdwoodsy) [inline Ju5ts0cRypt0 3 projects:14]: You need a comma here after the closing curly bracket.

Comment (Ju5ts0cRypt0) [reply Ju5ts0cRypt0 3 projects:14]: Do I need to resubmit? Sorry for the ignorance here.

Comment (weirdwoodsy) [reply Ju5ts0cRypt0 3 projects]: I'm nobody here. I just noticed an issue and thought I'd point it out.

But I don't think you will need a new PR. You already updated the JSON so you good.

**Comment by Ju5ts0cRypt0** [comment]
@JRypt0 on Twitter samredmixedmedia.com

**Pull request closed without merging**